apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/anyAssertions.kt

```diff
@@ -224,3 +227,32 @@ inline val <T> Expect<T>.it: Expect<T> get() : Expect<T> = this
  * @since 0.12.0
  */
 inline val <T> Expect<T>.its: Expect<T> get() : Expect<T> = this
+
+/**
+ * Expects that the subject of the assertion is not (equal to) [expected] and [otherValues].
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ *
+ * @since 0.13.0
+ */
+inline fun <reified T> Expect<T>.isNoneOf(expected: T, vararg otherValues: T): Expect<T> =
+    _logicAppend { isNotIn(expected glue otherValues) }
+
+/**
+ * Expects that the subject of the assertion is not (equal to) any value of [expected].
+ *
+ * Notice that a runtime check applies which assures that only [Iterable], [Sequence] or one of the [Array] types
+ * are passed. This function expects [IterableLike] (which is a typealias for [Any]) to avoid cluttering the API.
+ *
+ * @return An [Expect] for the current subject of the assertion.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ * @throws IllegalArgumentException in case the iterable is empty.
+ *
+ * @since 0.13.0
+ */
+inline fun <reified T> Expect<T>.isNotIn(expected: IterableLike): Expect<T> {
```

Comment:
```suggestion
infix fun <T> Expect<T>.isNotIn(expected: IterableLike): Expect<T> {
```
most important is that you add `infix` but I guess we don't need inline nor reified here.